UNRELEASED.md

```diff
@@ -12,6 +12,7 @@ Use [the changelog guidelines](https://git.io/polaris-changelog-guidelines) to f
 - Updated `Page` header layout so actions take up less room on small screens ([#707](https://github.com/Shopify/polaris-react/pull/707))
 - Added `alternateTool` prop to `ResourceList` component ([#812](https://github.com/Shopify/polaris-react/pull/812))
 - Added second thumb to `RangeSlider` ([#784](https://github.com/Shopify/polaris-react/pull/784))
+- Added clamp to `RangeSlider` so its initial value doesn't exceed min or max ([#834](https://github.com/Shopify/polaris-react/pull/834))
```

Comment:
I don’t think we need a changelog item for this

Reply:
On second thought, is it better to target master with this PR and add this changelog item as a bug fix?

Reply:
It can't target master just yet because it's relying on the changes in the dual thumb. But yes, I think this should be a bug fix. I am waiting to rebase with `range-slider-dual-input` when it has rebased with master.

Reply:
Yep, I’m suggesting refactoring this PR to fix the production bug

Reply:
We chatted about this in Slack and I'm going to stick with pointing it to the feature branch. The reasoning being that this bug is entirely dependent on a consumer error, so I don't consider it an urgent fix. The merge conflict would be a headache and outweighs the slight possibility that anyone would encounter this bug.

Reply:
Also, the reason I think we need a changelog entry is that it's fixing an existing component (the single thumb slider), whereas everything else targets the dual thumb, which is new.